src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -1,27 +1,64 @@
 import React from 'react';
+import { act } from 'react-dom/test-utils';
 import { mount } from 'enzyme';
-
+import * as auth from '@edx/frontend-platform/auth';
 import ProctoredExamSettings from './ProctoredExamSettings';
 
 const defaultProps = {
   courseId: 'course-v1%3AedX%2BDemoX%2BDemo_Course',
 };
 
+jest.mock('@edx/frontend-platform/auth');
```

Comment:
Have you tried testing it without this line? This line is already in setupTest.js

Reply:
Good feedback. Done